### ISSUE TITLE
US#36 Export articles data from prod

### ISSUE THREAD

**Issue by davidbgk** [cafebabel/cafebabel.com#194]
Refs https://github.com/cafebabel/cafebabel.com/issues/182

There look to be a memory issue due to the size of the generated file.

**Comment by davidbgk** [comment]
@vinyll maybe we should ask for octopus to help us?

**Comment by vinyll** [comment]
Mail sent

**Comment by davidbgk** [comment]
On prod side:

```
source ~/ve/bin/activate
./manage.py dumpdata --settings=settings.prod editorials.Base --indent=2 > base.json
./manage.py dumpdata --settings=settings.prod editorials.Article --indent=2 > articles.json
```

On local side:

```
rsync -az prod@91.194.60.65:~/cafebabel/base.json .
rsync -az prod@91.194.60.65:~/cafebabel/articles.json .
echo ] >> articles.json
```

**Comment by vinyll** [comment]
Articles are as follow in the old Django system:

```
Base (Abstract class)
  |- Article <- Revision
  |- Video
  |- Dossier
  |- Gallery
```

Next attempt is to make an export manually.

a dump of a recent database is available in mysql at `rsync -z --progress prod@91.194.60.65:/home/preprod/cafebabel-dump.20170218.sql .`
